content/source/docs/cloud/registry/using.html.md

```diff
@@ -65,7 +66,9 @@ Use the "Examples" dropdown to navigate to example modules and use the  Readme/I
 
 ![Terraform Cloud screenshot: a module submodules detail page](./images/using-module-examples.png)
 
-## Using Private Modules in Configurations
+## Using Modules in Configurations
+
+The syntax for referencing publicly curated modules in the [module block](/docs/language/modules/syntax.html) `source` argument is `<NAMESPACE>/<MODULE NAME>/<PROVIDER>`.
```

Comment:
Could we add an example underneath this one like there is for the private modules? I think that would help users!

Reply:
Yes.  It would be something like `hashicorp/subnets/cidr`